### ISSUE TITLE
[Lang]: add translation to `Brazilian Portuguese`

### ISSUE THREAD

**Issue by rafaelassumpcao** [aykutkardas/regexlearn.com#39]
I wish to translate it to portuguese if there is no problem to you @aykutkardas

i just havea question, on localization folder, i'm thinking in put the json as 'brazilian-portuguese' to diferentiate the portuguese from  portugal one its that ok?

**Comment by carlosbarretoeng** [comment]
I can help you @rafaelassumpcao. It's would be my first contribution to an open project. I don't know exactly how to work at this point.

**Comment by aykutkardas** [comment]
The text of some steps may change so I'll keep this issue open. I am definitely positive about the addition of local languages and I take it seriously. I hope those who want this language to be added send emoji here.

I will prepare a roadmap soon. 

Thank you for your interest. ✨  
Please stay tuned.

**Comment by rafaelassumpcao** [comment]
> I can help you @rafaelassumpcao. It's would be my first contribution to an open project. I don't know exactly how to work at this point.

Oh that's awesome, we can divide up some of the work, if you want and end up with 2 complementary PRs, but we should wait for the changes and the road map I think.

**Comment by carlosbarretoeng** [comment]
Perfect! Let's wait for the road map

**Comment by rafaelassumpcao** [comment]
Good to know @aykutkardas

So @carlosbarretoeng we can see we could share the remaining work to do. Do you have an idea?

**Comment by carlosbarretoeng** [comment]
I was thinking about it. 

Maybe we could translate each file in the 'localization' folder, each one of us one file, could work's. For control, before starting to translate, we inform here what file will be translated. What do you think about working in forked repositories? Or maybe can we work in the same repository as collaborators? Anyway, we should make a pull request (at least a full commit) for each file translated. This way, if occurs a mistake, would be easier to correct that.

I don't know if I made myself understand. What do you think about @aykutkardas  and @rafaelassumpcao ?

**Comment by aykutkardas** [comment]
As long as you're only sending the changed JSON files, it's okay to send them all in one PR or each file as a separate PR.

It is important not to submit a change to this [file](https://github.com/aykutkardas/regexlearn.com/blob/develop/src/localization/index.js) before all translations are complete.

I'm excited for a new language. I'll be here if you have any questions. ✨

**Comment by carlosbarretoeng** [comment]
Hi,

I submitted a PR #139 with a partial translation. Can you check, @rafaelassumpcao ?
